build.sbt

```diff
@@ -123,8 +124,8 @@ lazy val scala = Project(id = "content-atom-model", base = file("scala"))
       (scroogeUnpackDeps in Compile).value.flatMap { dir => (dir ** "*.thrift").get }
     },
     libraryDependencies ++= Seq(
-      "org.apache.thrift" % "libthrift" % "0.9.2",
-      "com.twitter" %% "scrooge-core" % "4.5.0"
+      "org.apache.thrift" % "libthrift" % "0.9.3",
+      "com.twitter" %% "scrooge-core" % "4.12.0"
```

Comment:
I am worrying a bit about those updates because I remembered [the incompatibility issue](https://github.com/guardian/content-api-models/pull/10) we encountered between `scrooge` (including `4.5.0`) and `libthrift` `0.9.2`.  It will be good if you could check if `scrooge` `4.12` is still compiled against `libthrift 0.5.0-1 (Twitter's fork)`.